applications/dashboard/models/class.usermodel.php

```diff
@@ -2932,18 +2932,24 @@ public function saveIP($userID, $IP, $dateUpdated = false) {
 
     /**
      * Updates visit level information such as date last active and the user's ip address.
-     *
-     * @param int $UserID
-     * @param string|int|float $ClientHour
+     * @param $UserID
+     * @param int|float $ClientHour
```

Comment:
Actually null is the same as not passing an argument in this case. It is not a "valid" parameter.